notebooks/03_categorical_pipeline_column_transformer.ipynb

```diff
@@ -367,7 +367,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from sklearn.experimental import enable_hist_gradient_boosting\n",
```

Comment:
Are we supposed to change the notebook? It should be automatically synchronized from the python files, isn't it?
I am worried that it would screw up something.